doc/external-commands.rst

```diff
@@ -3,7 +3,6 @@ External Commands
 
 Cabal provides a system for external commands, akin to the ones used by tools like ``git`` or ``cargo``.
```

Comment:
does `git` actually have external commands?

Reply:
AFAIU, Yes! https://git.github.io/htmldocs/howto/new-command.html

Reply:
As another example, here's how the cargo manual is redacted https://doc.rust-lang.org/cargo/reference/external-tools.html#custom-subcommands

Reply:
I have learned new stuff, thanks.